python/cudf/cudf/tests/test_parquet.py

```diff
@@ -3768,6 +3788,42 @@ def test_parquet_chunked_reader(
     assert_eq(expected, actual)
 
 
+@pytest.mark.parametrize("chunk_read_limit", [240])  # [0, 240, 102400000])
+@pytest.mark.parametrize("pass_read_limit", [240])  # [0, 240, 102400000])
+@pytest.mark.parametrize(
+    "nrows,skip_rows",
+    [
+        # These are OK
+        # Note: there is a bug if we read the buffer with read_parquet_chunked
+        # before cudf.read_parquet, where the cudf.read_parquet call will
+        # produce an invalid DF
```

Comment:
If you move the expected line before actual, these 3 tests should pass.

Not sure if this is a bug in libcudf or cudf-python, though.
(Looking at the buffers, they seem to be equal before and after the read_parquet_chunked call so maybe something strange is going on in cudf-python)

Reply:
I can't reproduce this one anymore after fixing the bug in cudf.read_parquet with nrows/skip_rows.

I suppose the segfault corrupted something in the buffers?